### ISSUE TITLE
LegendOption is missing a definition for property "icon"

### ISSUE THREAD

**Issue by Bilge** [apache/echarts#14690]
### Version
5.1.0

### Steps to reproduce
```js
chart.setOption(
    {
        legend: {
            icon: 'circle',
        },
    }
);
```

### What is expected?
No error.

### What is actually happening?
> TS2322: Type '{ icon: string; }' is not assignable to type 'Arrayable<LegendComponentOption> | undefined'.
  Object literal may only specify known properties, and 'icon' does not exist in type 'Arrayable<LegendComponentOption>'.

**Comment by plainheart** [comment]
Hi, #14263 fixed it in [5.1.0](https://github.com/apache/echarts/releases/tag/5.1.0).

**Comment by Bilge** [comment]
That seems like it would fix it, but I still have the problem in 5.1.0, so maybe it just hasn't been released yet?

**Comment by Ovilia** [comment]
Fixed in #14263
